tests/test_items.py

```diff
@@ -11,28 +11,28 @@
 from tests.test_case import TestCase
 
 TEST_ITEM = {
-    'item_id': '429994bf-784e-47cc-a823-e0c394b823e8',
-    'name': 'mario',
-    'price': 20.20,
-    'description': 'svariati mariii'
+    "item_id": "429994bf-784e-47cc-a823-e0c394b823e8",
+    "name": "mario",
+    "price": 20.20,
+    "description": "svariati mariii"
 }
 TEST_ITEM2 = {
-    'item_id': '577ad826-a79d-41e9-a5b2-7955bcf03499',
-    'name': 'GINO',
-    'price': 30.20,
-    'description': 'svariati GINIIIII'
+    "item_id": "577ad826-a79d-41e9-a5b2-7955bcf03499",
+    "name": "GINO",
+    "price": 30.20,
+    "description": "svariati GINIIIII"
 }
 TEST_ITEM_WRONG = {
-    'item_id': '19b4c6dc-e393-4e76-bf0f-72559dd5d32e',
-    'name': '',
-    'price': 30.20,
-    'description': 'svariati GINIIIII'
+    "item_id": "19b4c6dc-e393-4e76-bf0f-72559dd5d32e",
+    "name": "",
+    "price": 30.20,
+    "description": "svariati GINIIIII"
 }
 TEST_ITEM_PRECISION = {
-    'item_id': '68e587f7-3982-4b6a-a882-dd43b89134fe',
-    'name': 'Anna Pannocchia',
-    'price': 30.222222,
-    'description': 'lorem ipsum'
+    "item_id": "68e587f7-3982-4b6a-a882-dd43b89134fe",
+    "name": "Anna Pannocchia",
+    "price": 30.222222,
+    "description": "lorem ipsum"
```

Comment:
per questa volta per evitare che la pr rimanda sospesa a vita farò finta di non vedere questi cambiamenti di stile mischiati a quelli funzionali, ma dalla prossima volta dovranno essere in una pr separata.